hivemind/averaging/averager.py

```diff
@@ -554,7 +566,7 @@ def load_state_from_peers(self, wait=True) -> Optional[Tuple[Any, Sequence[torch
         """
         future = MPFuture()
         self._outer_pipe.send(("_load_state_from_peers", [], dict(future=future)))
-        return future.result() if wait else future
+        return future.result(timeout=timeout) if wait else future
```

Comment:
Is not it more practical to set the max time for loading the state _from one peer_?

I'm okay if this is addressed in a future PR.

Reply:
https://github.com/learning-at-home/hivemind/issues/372